docs/terms/validation_result.md

```diff
@@ -1,15 +1,91 @@
-----------
-A report generated from an expectation suite being run against a batch. The validation result is in JSON and is rendered as data docs.
\ No newline at end of file
+### Definition
+
+A Validation Result is an object generated when data is validated against an <TechnicalTag relative="../" tag="expectation" text="Expectation" /> or <TechnicalTag relative="../" tag="expectation_suite" text="Expectation Suite" />.
+
+### Features and promises
+
+Validation Results can be saved for every time that a <TechnicalTag relative="../" tag="checkpoint" text="Checkpoint" /> <TechnicalTag relative="../" tag="validation" text="Validates" /> data, permitting you to maintain records of your data's adherence to Expectations and track trends in the quality of the validated data.  Validation Results are also among the information passed to <TechnicalTag relative="../" tag="action" text="Actions" /> when a Checkpoint is run.  This allows them to be used for any purpose that an Action can be created to handle. 
+
+### Relationship to other objects
+
+Validation Results are generated inside a Checkpoint when a <TechnicalTag relative="../" tag="validator" text="Validator" /> runs an Expectation Suite against its paired <TechnicalTag relative="../" tag="batch_request" text="Batch Request" />.  The Validation Result will be passed to the Checkpoint's `action_list` where it will be available to any Actions that may need it.  The `StoreValidationResultAction` subclass of `ValidationAction` will save Validation Results to the <TechnicalTag relative="../" tag="validation_result_store" text="Validation Result Store" /> if it is present in a Checkpoint's `action_list`.
+
+## Use cases
+
+<CreateHeader/>
+
+A Validation Result is generated when you use [the interactive workflow to create Expectations](../guides/expectations/how_to_create_and_edit_expectations_with_instant_feedback_from_a_sample_batch_of_data.md) and the newly created Expectation is run against the sample <TechnicalTag relative="../" tag="batch" text="Batch" /> of data you are working with.  These individual Validation Results will be created as an `ExpectationValidationResult` instance.
+
+<ValidateHeader/>
+
+Validation Results are generated when you run Checkpoints to Validate data.  Because Checkpoints operate on Expectation Suites rather than individual Expectations, the Validation Results generated by Checkpoints will be created as `ExpectationSuiteValidationResult` instances. The difference between an `ExpectationValidationResult` instance and an `ExpectationSuiteValidationResult` instance is that the `ExpectationSuiteValidationResult` will contain additional information pertaining to the Expectation Suite that was Validated, and will contain the data of an `ExpectationValidationResult` for each Expectation in the Expectation Suite.  These `ExpectationValidationResult` instances will be in a list accessed by the `results` attribute of the `ExpectationSuiteValidationResult` instance.
```

Comment:
I think there might be a more straightforward way of saying this - basically an `ExpectationSuiteValidationResult` is to an `ExpectationValidationResult` as an `ExpectationSuite` is to an `Expectation`. One contains one more of the other (though technically a suite can have 0 Expectations), plus some additional metadata.

Reply:
I added this clarification and moved the more specific (and harder to parse) explanation into a followup paragraph.